### ISSUE TITLE
Assert implementation is correct

### ISSUE THREAD

**Issue by Veldhoen** [beeldengeluid/dane-visual-feature-extraction-worker#18]
Compare to Teng's example features, preferably in a unit test

**Comment by Veldhoen** [comment]
Took quite some deep debugging: first needed to implement proper indexing, to make sure we were comparing the right things.
Then we found out that model training mode needed to be swiched off, by calling `model.eval()`: training mode caused unexpected behaviour in the batch normalization layers
Also, we found that concatenating the feature tensors led to strange behaviour. We now first append all features to a list, and concatenate them to one big tensor in the end. 

On the way here, I realized data handling module should explicitly match keyframe and spectogram files, rather than just assuming they're both lists with the same length and contents: will make a new issue for this.
